### PR TITLE
Fix CI Dependency Check Failures

### LINKED ISSUE #20324: fix all ❌ and ⚠️ in `ci_check_dependency` (full TODO list)
Fix the last of existing issues that `ci_check_dependency` catches, so that devs can trust that if there are issues on their PRs, they need to address them before merging. 

Example PRs that have fixed other issues: 
* https://github.com/airbytehq/airbyte/pull/20274
* https://github.com/airbytehq/airbyte/pull/20110
* https://github.com/airbytehq/airbyte/pull/19865

A list of all the issues with the affected connectors can be found on [this draft PR](https://github.com/airbytehq/airbyte/pull/20306), where I edited the `ci_check_dependency` script to report on all existing connectors. **Please pull the latest master and merge it into the branch before working on it, maybe even again once your changes are approved so we don't miss anything.** If you work on that branch, make sure to remove the edits to the `ci_check_dependency` script (or just revert the first commit) before merging. We should then update the connector org about renewed expectations toward paying attention to this CI check.

Notes from @evantahler:
* we can ignore the source-scaffold-*  and destination-scaffold-* connectors
* we can (probably?) ignore source-relational-* connector too
* we can ignore source-python-http-tutorial

We can do the above by extending the current behavior around `IGNORED_DESTINATIONS`.

### PR DESCRIPTION
**Pull request by evantahler**
Closes airbytehq/airbyte#20324

Continuing the fine work in [#20274](https://github.com/airbytehq/airbyte/pull/20274), [#20110](https://github.com/airbytehq/airbyte/pull/20110), and [#19865](https://github.com/airbytehq/airbyte/pull/19865), this PR once again attempts to add missing documentation and metadata connectors need. 

**AS OF THIS PULL REQUEST, EVERY CONNECTOR HAS A DOC, CHANGELOG, AND IS PROPERLY PUBLISHED**
_Except..._

## Notes:

These problems will remain after this PR, and should be addressed by the connector contributors:

### Google Analytics

There's something very strange going on with the Google Analytics Sources:
* The connector named [`source-google-analytics-data-api`](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-google-analytics-data-api/Dockerfile) is using the doc called `sources/google-analytics-v4.md` (v0.0.3 is current)
* The connector named [`source-google-analytics-v4`](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-google-analytics-v4/Dockerfile) is using the doc called `sources/google-analytics-universal-analytics` (v0.1.32 is current)

Link to [slack thread](https://airbytehq-team.slack.com/archives/C037T53V0SJ/p1662480884997719?thread_ts=1662479535.580149&cid=C037T53V0SJ) about latest converstation and an issue (https://github.com/airbytehq/airbyte-internal-issues/issues/771) with more information

### Mongo

I don't really understand what happened with MongoDB.  There's a "v2" connector (not v2 of the version) 

### source-yahoo-finance-price

Not published.  We should publish this connector.

### destination-kvdb

Not Published

### destination-mysql-strict-encrypt

Version doesn't match `destination-mysql`

### CODE REVIEW

**Comment by evantahler** [comment]
Note: There's something very strange going on with the Google Analytics Sources:
* The connector named `source-google-analytics-data-api` is using the doc called `sources/google-analytics-v4.md`
* The connector named `source-google-analytics-v4` is using the doc called `sources/google-analytics-universal-analytics`

@Amruta-Ranade Would it be OK if I renamed the doc files so that they matched the names of the connectors?

File: docs/integrations/sources/dixa.md [hunk 1]
```diff
@@ -49,9 +49,9 @@ When using the connector, keep in mind that increasing the `batch_size` paramete
...
 
-| Version | Date | Pull Request | Subject |
-| :--- | :--- | :--- | :--- |
-| 0.1.2 | 2021-11-08 | [7499](https://github.com/airbytehq/airbyte/pull/7499) | Remove base-python dependencies |
-| 0.1.1 | 2021-08-12 | [5367](https://github.com/airbytehq/airbyte/pull/5367) | Migrated to CI Sandbox, refactorred code structure for future support |
-| 0.1.0 | 2021-07-07 | [4358](https://github.com/airbytehq/airbyte/pull/4358) | New source |
-
+| Version | Date       | Pull Request                                             | Subject                                                               |
+| :------ | :--------- | :------------------------------------------------------- | :-------------------------------------------------------------------- |
+| 0.1.3   | 2021-07-07 | [14437](https://github.com/airbytehq/airbyte/pull/14437) | 🎉 Source Dixa: bump version 0.1.3                                    |
```
Comment (alafanechere) [inline docs/integrations/sources/dixa.md:54]:
```suggestion
| 0.1.3   | 2022-07-07 | [14437](https://github.com/airbytehq/airbyte/pull/14437) | 🎉 Source Dixa: bump version 0.1.3                                    |
```

File: docs/integrations/sources/kustomer-singer.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+## Requirements
+
+- **Kustomer API token**. See the [Kustomer docs](https://help.kustomer.com/api-keys-SJs5YTIWX) for information on how to obtain an API token.
+
+## Changelog
+
+| Version | Date       | Pull Request                                           | Subject                                                                           |
+| :------ | :--------- | :----------------------------------------------------- | :-------------------------------------------------------------------------------- |
+| 0.1.2   | 2021-12-05 | [8578](https://github.com/airbytehq/airbyte/pull/8578) | Update fields in source-connectors specifications                                 |
```
Comment (alafanechere) [inline docs/integrations/sources/kustomer-singer.md:45]:
```suggestion
| 0.1.2   | 2021-12-25 | [8578](https://github.com/airbytehq/airbyte/pull/8578) | Update fields in source-connectors specifications                                 |
```

File: docs/integrations/sources/kustomer-singer.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+| 0.1.1   | 2021-12-13 | [8738](https://github.com/airbytehq/airbyte/pull/8738) | Deleted `user-agent`, `date_window_size`, `page_size_limit` from `spec.json` file |
```
Comment (alafanechere) [inline docs/integrations/sources/kustomer-singer.md:46]:
```suggestion
| 0.1.1   | 2021-12-22 | [8738](https://github.com/airbytehq/airbyte/pull/8738) | Deleted `user-agent`, `date_window_size`, `page_size_limit` from `spec.json` file |
```

File: docs/integrations/sources/pardot.md [hunk 4]
```diff
@@ -4,6 +4,7 @@ The Airbyte Source for [Salesforce Pardot](https://www.pardot.com/)
 
 ## Changelog
 
-| Version | Date       | Pull Request                                           | Subject            |
-| :------ | :--------- | :----------------------------------------------------- | :----------------- |
-| 0.1.0   | 2021-11-09 | [7091](https://github.com/airbytehq/airbyte/pull/7091) | New Source: Pardot |
+| Version | Date       | Pull Request                                             | Subject               |
+| :------ | :--------- | :------------------------------------------------------- | :-------------------- |
+| 0.1.1   | 2021-10-16 | [19618](https://github.com/airbytehq/airbyte/pull/19618) | Fix `visitors` stream |
```
Comment (alafanechere) [inline docs/integrations/sources/pardot.md:9]:
```suggestion
| 0.1.1   | 2022-12-16 | [19618](https://github.com/airbytehq/airbyte/pull/19618) | Fix `visitors` stream |
```

File: docs/integrations/sources/pardot.md [hunk 5]
```diff
@@ -4,6 +4,7 @@ The Airbyte Source for [Salesforce Pardot](https://www.pardot.com/)
...
+| 0.1.0   | 2021-10-16 | [7091](https://github.com/airbytehq/airbyte/pull/7091)   | 🎉 New Source: Pardot |
```
Comment (alafanechere) [inline docs/integrations/sources/pardot.md:10]:
```suggestion
| 0.1.0   | 2021-11-19 | [7091](https://github.com/airbytehq/airbyte/pull/7091)   | 🎉 New Source: Pardot |
```

File: docs/integrations/sources/yahoo-finance-price.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# Yahoo Finance Price
+
+The Airbyte Source for [Yahoo Finance Price](https://finance.yahoo.com/)
+
+## Changelog
+
+| Version | Date       | Pull Request                                             | Subject                       |
+| :------ | :--------- | :------------------------------------------------------- | :---------------------------- |
+| 0.1.3   | 2022-03-23 | [10563](https://github.com/airbytehq/airbyte/pull/10563) | 🎉 Source Yahoo Finance Price |
```
Comment (alafanechere) [inline docs/integrations/sources/yahoo-finance-price.md:9]: This connector is not published to docker hub. I think we should mention it in the doc.

File: tools/bin/ci_check_dependency.py [hunk 7]
```diff
@@ -17,9 +17,20 @@
     "/integration_tests/", "/unit_tests/",
     # Common
     "acceptance-test-config.yml", "acceptance-test-docker.sh", ".md", ".dockerignore", ".gitignore", "requirements.txt"]
+IGNORED_SOURCES = [
+    re.compile("^source-e2e-test-cloud$"),
+    re.compile("^source-mongodb$"),
```
Comment (alafanechere) [inline tools/bin/ci_check_dependency.py:22]: FYI `source-mongodb-v2` is the maintained MongoDb source. The other one is legacy.

**Review by alafanechere — APPROVED** [review]
🧹 Thanks for this chore! 
I spotted a couple of changelog inconsistencies.

**Comment by octavia-approvington** [comment]
Enough! Let's do this <br/> goooooo!

**Review by octavia-approvington — APPROVED** [review]

**Pull request merged**